### PR TITLE
Ensures that when PULFA ARKs are encountered they are not updated

### LINKED ISSUE #2595: Refuse to update PULFA ARKs, but don't error
We currently raise an exception when refusing to update PULFA ARKs:

https://github.com/pulibrary/figgy/blob/master/app/services/identifier_service.rb#L28

But we should notify the user that the ARK won't be updated, but still allow saving and other actions to work.

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #2595

### CODE REVIEW

File: app/controllers/concerns/resource_controller.rb [hunk 1]
```diff
@@ -59,21 +59,9 @@ def edit
   end
 
   def update
-    @change_set = change_set_class.new(find_resource(params[:id])).prepopulate!
+    @change_set = new_change_set.prepopulate!
     authorize! :update, @change_set.resource
-    if @change_set.validate(resource_params)
```
Comment (tpendragon) [inline app/controllers/concerns/resource_controller.rb]: I think these extractions make this method harder to parse. What are other's thoughts?

Comment (escowles) [reply app/controllers/concerns/resource_controller.rb]: I find this diff very hard to read, but looking at the end state, the `update` method looks very clean and simple to me.

I'm not sure extracting the `change_set` and `new_change_set` methods is worth it (in fact, I couldn't find any place that's using the new `change_set` method which line 62 duplicates).

Comment (hackmastera) [reply app/controllers/concerns/resource_controller.rb]: Yeah I guess it feels more straightforward to have the logic in the update action if it fits. Did rubocop complain?

Comment (jrgriffiniii) [reply app/controllers/concerns/resource_controller.rb]: Not about this restructuring, but I can return to the original form of the `#update` action.

**Pull request merged**